src/app/search/components/lists/list.component.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import { ReactElement, useContext } from "react";
+
+import Link from "next/link";
+
+import Image from "next/image";
+
+import MingcuteBookmarkLine from "@/icons/MingcuteBookmarkLine";
+
+import { JobsContext } from "@/app/search/providers/jobs/jobs.provider";
+
+import styles from "./list.module.css";
+
+export default function ListsComponent(): ReactElement {
+  const { filteredJobs } = useContext(JobsContext);
+
+  return (
+    <ul className={styles.lists}>
+      {filteredJobs.map((job) => (
+        <li key={job.id}>
+          <div className={styles.header}>
```

Comment:
به جای این کار بهتره از خودِ تگِ header استفاده کنید

```suggestion
          <header>
```

Reply:
اصلاح شد